node/src/components/consensus/era_supervisor.rs

```diff
@@ -93,6 +93,8 @@ pub struct EraSupervisor<I> {
     /// A node keeps `2 * bonded_eras` past eras around, because the oldest bonded era could still
     /// receive blocks that refer to `bonded_eras` before that.
     bonded_eras: u64,
+    /// The height of the next block to be finalized
+    next_block_height: u64,
```

Comment:
Do you think it would be a worthwhile to comment why this field is here? It's not immediately obvious. For example, `bonded_eras` has a comment explaining why we need it.